modules/core/05-port/types/ibc_legacy_module.go

```diff
@@ -45,9 +45,8 @@ func (im *LegacyIBCModule) OnChanOpenInit(
 	counterparty channeltypes.Counterparty,
 	version string,
 ) (string, error) {
-	negotiatedVersions := make([]string, len(im.cbs))
-
-	for i := len(im.cbs) - 1; i >= 0; i-- {
+	var negotiatedVersions []string
```

Comment:
[nit] we could be slightly more efficient by doing:

`negotiatedVersions := make([]string,0, len(im.cbs))` which would pre-allocate the needed capacity (but not the length).

If we know there's a 1:1 relationship between `im.cbs` and `negotiatedVersions` (so that the length is exactly the same) we could even do 

```go
negotiatedVersions := make([]string,len(im.cbs)

for i, cb := range im.reverseCallbacks() {
     ...
     negotiatedversions[i] =

```

Reply:
yeah no harm!